src/actions/__tests__/eval-frame-action-validator.test.js

```diff
@@ -28,7 +28,7 @@ describe("validateActionFromEvalFrame should throw errors as expected", () => {
     expect(() =>
       validateActionFromEvalFrame({
         type: "UPDATE_VALUE_IN_HISTORY",
-        historyId: 12424242
+        historyItem: 10
```

Comment:
this test should probably be expanded a since `historyItem` is now a composite object -- we'll want to test that the things sent over within historyItem are of the correct type, not just that historyItem is an object and not an integer

Reply:
i.e., in addition to this test case, we should add cases for subfields

Reply:
yep good point, happy to add tests here

Reply:
hmm, on second thought, this isn't really the point of this test suite, is it? It looks like it's meant to test the action validator, not the contents of the eval frame schema. I added one additional test to this but it doesn't really feel like it belongs.